api/Entity.go

```diff
@@ -159,6 +159,22 @@ func (vrsConnection *VRSConnection) GetEntityPorts(uuid string) ([]string, error
 	return ovsdb.UnMarshallOVSStringSet(row[ovsdb.NuageVMTableColumnPorts])
 }
 
+// GetEntityName retrieves entity name from OVSDB
+func (vrsConnection *VRSConnection) GetEntityName(uuid string) (string, error) {
```

Comment:
Just an idea. Is it better if we have a function `GetEntityInfo` which returns `EntityInfo` struct defined in this file?
 We have two functions `GetEnitityPorts` and `GetEntityName` now. I feel it is better to have one generic function than having multiple specific functions. Later we may need some other info related to entity. We can keep `GetEntityPorts` to ensure backwards compatiability.